arrays/victor/C#_soft-188/1_task/1_task/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace _1_task
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] mas = { 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270 };
+            foreach (var v in mas)
+                Console.Write (v + " ");
+                Console.WriteLine();
+                
+            Array.Reverse(mas);
```

Comment:
In this task you can't output the elements of array using C# methods. Please write your own Reverse() method